README.md

```diff
@@ -3,74 +3,204 @@
+variables. If set to `no`, all HA Cluster configuration will be purged from
+target hosts.
+
+#### `ha_cluster_start_on_boot`
+
+boolean, default: `yes`
+
+If set to `yes`, cluster services will be configured to start on boot. If set
+to `no`, cluster services will be configured not to start on boot.
+
+#### `ha_cluster_fence_agent_packages`
+
+list of fence agent packages to install, default: fence-agents-all, fence-virt
+
+#### `ha_cluster_hacluster_password`
+
+string, no default - must be specified
+
+Password of the `hacluster` user. This user has full access to a cluster.
+
+#### `ha_cluster_corosync_key_src`
+
+path to corosync authkey file, default: `null`
+
+Authentication and encryption key for Corosync communication. It is highly
+recommended to have a unique value for each cluster. The key should be 256
+bytes of random data.
+
+If no key is specified, a key already present on the nodes will be used. If
+nodes don't have the same key, a key from one node will be distributed to other
+nodes so that all nodes have the same key. If no node has a key, a new key will
+be generated and distributed to the nodes.
+
+If this variable is set, `ha_cluster_regenerate_keys` is ignored for this key.
+
+#### `ha_cluster_pacemaker_key_src`
+
+path to pacemaker authkey file, default: `null`
+
+Authentication and encryption key for Pacemaker communication. It is highly
+recommended to have a unique value for each cluster. The key should be 256
+bytes of random data.
+
+If no key is specified, a key already present on the nodes will be used. If
+nodes don't have the same key, a key from one node will be distributed to other
+nodes so that all nodes have the same key. If no node has a key, a new key will
+be generated and distributed to the nodes.
+
+If this variable is set, `ha_cluster_regenerate_keys` is ignored for this key.
+
+#### `ha_cluster_fence_virt_key_src`
+
+path to fence-virt or fence-xvm pre-shared key file, default: `null`
+
+Authentication key for fence-virt or fence-xvm fence agent.
+
+If no key is specified, a key already present on the nodes will be used. If
+nodes don't have the same key, a key from one node will be distributed to other
+nodes so that all nodes have the same key. If no node has a key, a new key will
+be generated and distributed to the nodes.
+
+If this variable is set, `ha_cluster_regenerate_keys` is ignored for this key.
+
+If you let the role to generate new key, you are supposed to copy the key to
+your nodes' hypervisor to ensure that fencing works.
+
+#### `ha_cluster_pcsd_public_key_src`, `ha_cluster_pcsd_private_key_src`
+
+path to pcsd TLS certificate and key, default: `null`
+
+TLS certificate and private key for pcsd. If this is not specified, a
+certificate - key pair already present on the nodes will be used. If
+certificate - key pair is not present, a random new one will be generated.
+
+If these variables are set, `ha_cluster_regenerate_keys` is ignored for this
+certificate - key pair.
```

Comment:
add text like "If private key value is provided, recommend vault encrypting the value. See https://docs.ansible.com/ansible/latest/user_guide/vault.htm"

Reply:
fixed in 1c2a9bf